docs/source/locking.rst

```diff
@@ -42,7 +53,7 @@ The server responds with status :term:`200 OK` and returns the lock information.
 Unlocking an object
 -------------------
 
-To unlock an object send a POST request to the ``/@unlock`` endpoint.
```

Comment:
@avoinea shall we add a section here for the deprecated methods that we have and that we will drop in plone.restapi 9?

Reply:
Added.